components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -538,6 +545,7 @@ export class GitpodServerImpl<Client extends GitpodClient, Server extends Gitpod
         // that is logged in).
         // The guard check happens in guardAdminAccess(...) for admin users.
         if (!admin) {
+            // TODO(janx): allow team
```

Comment:
In the current PR state, I expect that `stopWorkspace` works for team members, but then `internalStopWorkspace` fails -- still TODO